src/test/java/uk/gov/ida/verifylocalmatchingserviceexample/service/Cycle1MatchingServiceTest.java

```diff
@@ -11,218 +14,180 @@
-        MatchingAttributesValueDto verifiedDateOfBirth = getVerifiedDateOfBirth(dateOfBirth);
-        cycle1MatchingService = new Cycle1MatchingService(personDAO, verifiedPidDAO);
-        LinkedList<Person> matchingUsers = new LinkedList<Person>() {{
-            add(new Person(verifiedSurnames.get(0), dateOfBirth, 13332));
-            add(new Person(verifiedSurnames.get(1), dateOfBirth.plusDays(3), 1233));
-        }};
-        when(personDAO.getMatchingUsers(verifiedSurnames, dateOfBirth)).thenReturn(matchingUsers);
-
-        MatchingServiceRequestDto requestWithVerifiedSurnames = aMatchingServiceRequestDtoBuilder()
-                .withMatchingAttributesDto(aMatchingAttributesDtoBuilder()
-                        .withSurnames(getVerifiedSurnames((String[]) verifiedSurnames.toArray()))
-                        .withDateOfBirth(verifiedDateOfBirth)
-                        .build())
-                .build();
-
-        MatchStatusResponseDto matchStatusResponseDto = cycle1MatchingService.matchUser(requestWithVerifiedSurnames);
-
-        assertThat(matchStatusResponseDto).isEqualTo(MatchStatusResponseDto.NO_MATCH);
+    public void shouldReturnNoMatchWhenMultipleRecordsMatchSurnameDateOfBirthAndPostcode() {
+        LocalDate dateOfBirth = LocalDate.now().minusYears(10);
+        Person personOne = aPerson()
+            .withSurname("some-surname")
+            .withDateOfBirth(dateOfBirth)
+            .withAddress(anAddress().withPostcode("some-postcode").build())
+            .build();
+        Person personTwo = aPerson()
+            .withSurname("some-surname")
+            .withDateOfBirth(dateOfBirth)
+            .withAddress(anAddress().withPostcode("some-postcode").build())
+            .build();
+        MatchingAttributesValueDto surnameAttribute = aMatchingAttributesValueDtoBuilder().withValue("some-surname").withVerified(true).build();
+        MatchingAttributesValueDto dateOfBirthAttribute = aMatchingAttributesValueDtoBuilder().withValue(dateOfBirth).withVerified(true).build();
+        AddressDto address = anAddressDtoBuilder().withPostCode("some-postcode").withVerified(true).build();
+
+
+        when(matchingServiceRequestDto.getMatchingAttributesDto().getSurnames()).thenReturn(Arrays.asList(surnameAttribute));
+        when(matchingServiceRequestDto.getMatchingAttributesDto().getDateOfBirth()).thenReturn(dateOfBirthAttribute);
+        when(matchingServiceRequestDto.getMatchingAttributesDto().getAddresses()).thenReturn(Arrays.asList(address));
+        when(personDAO.getMatchingUsers(any(), any())).thenReturn(Arrays.asList(personOne, personTwo));
+
+        assertThat(cycle1MatchingService.matchUser(matchingServiceRequestDto)).isEqualTo(NO_MATCH);
     }
 
-    @Test
-    public void shouldSaveVerifiedPidInTheDatabaseWhenCycle1MatchIsFound() {
-        String surname = "test-surname";
-        List<String> surnames = singletonList(surname);
-        LocalDate dateOfBirth = LocalDate.of(1993, 8, 16);
```

Comment:
Is there particular reason why these tests are removed ?

Reply:
I must have removed it by mistake when mergeing. Will add it again.